account_move_line_sale_info/tests/test_account_move_line_sale_info.py

```diff
@@ -0,0 +1,246 @@
+# Copyright 2020 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo.tests import common
+
+
+class TestAccountMoveLineSaleInfo(common.TransactionCase):
+    def setUp(self):
+        super(TestAccountMoveLineSaleInfo, self).setUp()
+        self.sale_model = self.env["sale.order"]
+        self.sale_line_model = self.env["sale.order.line"]
+        self.invoice_model = self.env["account.move"]
+        self.invoice_line_model = self.env["account.move.line"]
```

Comment:
```suggestion
        self.am_model = self.env["account.move"]
```
line model is repeated